packages/runner/lib/sdk/sdk.unit.test.ts

```diff
@@ -151,50 +145,44 @@ describe('Pagination', () => {
             token_url: '',
             docs: ''
         };
-        (await import('@nangohq/node')).Nango.prototype.getIntegration = vi.fn().mockReturnValue({ config: { provider: 'github' } });
-        vi.spyOn(providerService, 'getProvider').mockImplementation(() => provider);
+        vi.spyOn(await import('@nangohq/shared-public'), 'getProvider').mockImplementation(() => provider);
```

Comment:
Since dryRun is no longer available in the backend, I had to change the mocking strategy, thankfully it was not too bad